### DISCUSSION TITLE
Add Feed

### DISCUSSION THREAD

**Discussion by krzysztofnowak** [dailydotdev/daily#908] [Feature Requests]
Add Feed in RSS style or anything else. It would be helpful to include articles into Brave browser news

**Comment by idoshamun** [comment]
Hi @krzysztofnowak,

Thanks for the suggestion. Supporting RSS for the feed is currently out of scope but we support RSS for bookmarks. We are very much focused on the experience you get from within the platform.
So you can enable RSS integration for your bookmarks and plug it to every other system
